docs/development_guide.md

```diff
@@ -51,7 +55,56 @@ Note that after running the above command we get some dependency conflicts in
 installed packages which is currently safe to ignore. For details see
 [Issue #71](https://github.com/googlegenomics/gcp-variant-transforms/issues/71).
 
+### Setup IDE
+
+You may choose any IDE as you like. The following steps are intended for
+IntelliJ users.
+
+#### Install IntelliJ IDE
+
+Download
+[IntelliJ IDEA Community Edition](https://www.jetbrains.com/idea/download/#section=linux)
+and install.
+
+#### Install Python plugin
+
+Choose File | Settings on the main menu, and then go to Plugins.
+Click the Install JetBrains plugin button.
+In the dialog that opens, search for Python Community Edition and then install
+the plugin.
+
+For more details, refer to
+[Install plugins](https://www.jetbrains.com/help/idea/installing-updating-and-uninstalling-repository-plugins.html).
+
+#### Setup IntelliJ SDK
+
+1. Choose File | Project Structure on the main menu, and then go to Project.
+2. Create a new project SDK by clicking the New button, choose Python SDK then
+add Local.
+3. In the dialog that opens, click the Virtual Environment node. Select New
+environment, and specify the location of the new virtual environment. Note that
+the folder where the new virtual environment should be located must be empty!
+For the Base interpreter, add the python path
+gcp-variant-transforms/venv/bin/python under the created virtualenv.
+
+#### Code Inspection
+
+The inspection profile in .idea/inspectionProfiles/Project_Default.xml is
+checked into the git repository and can be imported into
+File | Settings | Editor | Inspections.
+
+Code inspections can be run from the Analyze menu. The result window can be
+accessed from View > Tool Windows.
```

Comment:
Should you add any comments about code inclusion/exclusion? I have excluded everything other than gcp_variant_transforms (see screenshot) and IIRC this was important to trim the long list of inspection warnings.
![project_structure](https://user-images.githubusercontent.com/11869543/36817198-47274e20-1cae-11e8-998c-4a6fd8a2bb07.png)

Reply:
Done.